charts/pulsar/templates/pulsar-manager-service.yaml

```
@@ -33,6 +33,9 @@ spec:
   {{- if .Values.pulsar_manager.service.externalTrafficPolicy }}
   externalTrafficPolicy: {{ .Values.pulsar_manager.service.externalTrafficPolicy }}
   {{- end }}
+  {{- with .pulsar_manager.service.loadBalancerClass }}
```

```suggestion
  {{- with .Values.pulsar_manager.service.loadBalancerClass }}
```
fixes the invalid reference